examples/llm/rag/standalone_pipeline.py

```diff
@@ -48,9 +48,8 @@ def _build_engine(model_name: str, vdb_resource_name: str, llm_service: str, emb
 
     prompt = """You are a helpful assistant. Given the following background information:\n
 {% for c in contexts -%}
-Title: {{ c.title }}
-Summary: {{ c.summary }}
-Text: {{ c.page_content }}
+Header: {{ c.header }}
+Data: {{ c.data }}
```

Comment:
This will not be backwards compatible. We will need to modify this to work with a variable number of options. This should work (Please test to make sure it functions as Im expecting):
```suggestion
{% if c.title %}
Title: {{ c.title }}
{% endif %}
{% if c.summary %}
Summary: {{ c.summary }}
{% endif %}
{% if c.page_content %}
Text: {{ c.page_content }}
{% endif %}
{% if c.header %}
Header: {{ c.header }}
{% endif %}
{% if c.data %}
Data: {{ c.data }}
{% endif %}
```